test/e2e-secondary-network/secondary_network_test.go

```diff
@@ -223,6 +226,146 @@ func (data *testData) pingBetweenInterfaces(t *testing.T) error {
 	return nil
 }
 
+// getIPPoolNames retrieves the IPPool names associated with a given set of interfaces in a Pod. It queries the NetworkAttachmentDefinition for each interface
```

Comment:
I feel you comments are too long. We generally wrap comments at 80. You may also do 100, 120, but not too long.

Reply:
you didn't address this comment @KMAnju-2021 
it seems the line is still 150+ chars